dashboard/src/app/workspaces/workspace-details/environments/list-env-variables/edit-variable-dialog/edit-variable-dialog.controller.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2015-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+'use strict';
+
+/**
+ * @ngdoc controller
+ * @name list.environment.variables.controller:EditVariableDialogController
+ * @description This class is handling the controller for the dialog box about editing the environment variable.
+ * @author Oleksii Kurinnyi
+ */
+export class EditVariableDialogController {
+
+  /**
+   * Default constructor that is using resource
+   * @ngInject for Dependency injection
+   */
+  constructor($mdDialog) {
+    this.$mdDialog = $mdDialog;
+
+    this.updateInProgress = false;
+    this.name = this.envVariableName;
+    this.value = this.envVariableValue;
+  }
```

Comment:
why do we need to introduce this.name and this.value if we already have this.envVariableName and this.envVariableValue ?